css/custom.css

```diff
@@ -3894,7 +3894,7 @@ img.emoji {
 }
 
 .tabPanel-widget h2 {
-    font-size: 1.4rem;
+    font-size: 1rem;
```

Comment:
If you're going to change both values to same 1rem you don't have to include this line (because @ media already won't change anything)

Reply:
anyways it can stay at this moment i can clear up this and other codes later

Reply:
#2324 related